cmake/EkatCreateUnitTest.cmake

```diff
@@ -4,7 +4,7 @@ include(EkatUtils) # To check macro args
 # Note: we have to set this variable here, so CMAKE_CURRENT_LIST_DIR gets the
 #       directory of this file. If we did it inside the function, it would get
 #       the directory from where the function is called
-set(CATCH_INCLUDE_DIR ${CMAKE_CURRENT_LIST_DIR}/../extern/catch2/include)
+set(CATCH_INCLUDE_DIR ${CMAKE_CURRENT_LIST_DIR}/../extern/Catch2/src)
```

Comment:
I don't think we need this var anymore. Linking against catch2 library, built within this cmake project, should already populate with Catch2 include paths. And I saw that their `src` folder is included in the PUBLIC section of their library.

Reply:
This is just me not understanding all the assumptions made by our build system. Happy to remove it.